index.html

```diff
@@ -51,6 +51,166 @@ <h2 class="color-green">I'm a software developer</h2>
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <div class="buttons">
+                  <div>Ruby on rails</div>
+                  <div>css</div>
+                  <div>JavScript</div>
+                  <div>html</div>
+                </div>
+                <div class="button-link">
+                  <button class="button" type="button">See Project</button>
+                </div>
+              </div>
+            </li>
+            <li class="list-item">
+              <div class="image-placeholder"></div>
+              <div class="image-desc">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <div class="buttons">
+                  <div>Ruby on rails</div>
+                  <div>css</div>
+                  <div>JavScript</div>
+                  <div>html</div>
+                </div>
+                <div class="button-link">
+                  <button class="button" type="button">See Project</button>
+                </div>
+              </div>
+            </li>
+            <li class="list-item">
+              <div class="image-placeholder"></div>
+              <div class="image-desc">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <div class="buttons">
+                  <div>Ruby on rails</div>
+                  <div>css</div>
+                  <div>JavScript</div>
+                  <div>html</div>
+                </div>
+                <div class="button-link">
+                  <button class="button" type="button">See Project</button>
+                </div>
+              </div>
+            </li>
+            <li class="list-item">
+              <div class="image-placeholder"></div>
+              <div class="image-desc">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <div class="buttons">
+                  <div>Ruby on rails</div>
+                  <div>css</div>
+                  <div>JavScript</div>
+                  <div>html</div>
+                </div>
+                <div class="button-link">
+                  <button class="button" type="button">See Project</button>
+                </div>
+              </div>
+            </li>
+            <li class="list-item">
+              <div class="image-placeholder"></div>
+              <div class="image-desc">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <div class="buttons">
+                  <div>Ruby on rails</div>
+                  <div>css</div>
+                  <div>JavScript</div>
+                  <div>html</div>
+                </div>
+                <div class="button-link">
+                  <button class="button" type="button">See Project</button>
+                </div>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <div class="about-me">
+        <section class="about-container">
+          <header class="about-header">
+            <h1>About me</h1>
```

Comment:
- [ ] The `h1` tag is reserved to be used only once and as a title to the website. Kindly change this tag to an `h2` tag.